src/mongo/MongoCommands.ts

```diff
@@ -18,16 +18,61 @@ import { MongoFindOneResultEditor } from './editors/MongoFindOneResultEditor';
 import { MongoCommand } from './MongoCommand';
 import { MongoDatabaseTreeItem } from './tree/MongoDatabaseTreeItem';
 
+const output = vscodeUtil.getOutputChannel();
+const notInScrapbookMessage = "This command can only be run inside of a MongoDB scrapbook (*.mongo)";
+
 export class MongoCommands {
+
+	public static async executeAllCommandsFromActiveEditor(database: IAzureParentNode<MongoDatabaseTreeItem>, extensionPath, editorManager: CosmosEditorManager, tree: AzureTreeDataProvider, context: IActionContext): Promise<void> {
```

Comment:
FWIW I agree it's generally better to keep functional changes separate from refactoring, but I make an exception when people clean up old code that they're actively changing anyways. It feels like a sustainable way to clean up code as we go rather than filing a bunch of engineering issues. No strong opinion, though - separate PR is fine.